SECURITY.md

```diff
@@ -32,12 +32,22 @@ As these 3rd party audits are completed and issues are sufficiently addressed, w
   - **Scope**: _pyth-crosschain-evm contracts_
 - **[December 13, 2022 - CertiK](https://github.com/pyth-network/audit-reports/blob/pyth-certik/2022_12_13/pyth-crosschain-governance.pdf)**
   - **Scope**: _pyth-crosschain-governance contracts_
+- **[December 12, 2022 - Ottersec](https://github.com/pyth-network/audit-reports/blob/main/2023_27_07/pyth_sui_audit_final.pdf)**
+  - **Scope**: _pyth-crosschain-sui contracts_
 - **[December 13, 2022 - CertiK](https://github.com/pyth-network/audit-reports/blob/pyth-certik/2022_12_13/pyth-crosschain-solana.pdf)**
   - **Scope**: _pyth-crosschain-solana contracts_
 - **[February 23, 2023 - CertiK](https://github.com/pyth-network/audit-reports/blob/pyth-certik/2023_02_23/pyth-crosschain-eth.pdf)**
   - **Scope**: _pyth-crosschain-evm contracts_
 - **[March 14, 2023 - Zellic](https://github.com/pyth-network/audit-reports/blob/main/2023_03_14/Pyth%20Network%20-%20Zellic%20Audit%20Report.pdf)**
   - **Scope**: _pyth-crosschain-cosmwasm contracts_
+- **[July 10, 2023 - Zellic](https://github.com/pyth-network/audit-reports/blob/main/2023_26_07/EVM%20Patch%20Review.pdf)**
+  - **Scope**: _pyth-crosschain-evm contracts_
+- **[July 24, 2023 - Zellic](https://github.com/pyth-network/audit-reports/blob/main/2023_26_07/Pyth%20Network%20Smart%20Contract%20Patch%20Review.pdf)**
+  - **Scope**: _pyth-crosschain-cosmwasm/aptos/sui contracts_
+- **[July 31, 2023 - Trail of Bits](https://github.com/pyth-network/audit-reports/blob/main/2023_07_31/Trail%20of%20Bits%20Pythnet%20Report.pdf)**
+  - **Scope**: _pyth-crosschain xc_frontend, message buffer/remote executor/merkel tree library contracts_
+- **[January 18, 2024 - Trail of Bits](https://github.com/pyth-network/audit-reports/blob/main/2024_23_01/Pyth%20Data%20Association%20-%20Entropy%20-%20Comprehensive%20Report.pdf)**
+  - **Scope**: _pyth-crosschain-entropy contracts and fortuna web service_
 
```

Comment:
Can you add ottersec too https://github.com/pyth-network/audit-reports/tree/main/2024_04_25